README.md

```diff
@@ -137,6 +137,7 @@ Several options may be configured globally per-app.
 
 | Property           | Type              | Default                | Description |
 |--------------------|-------------------|------------------------|-------------|
+|`filterFields`      |`string`           |`""`                    |A comma separated string that contains the names of the properties to which the filter will be applied.  _(This can be overridden per-table)_|
```

Comment:
I don't think `filterfields` added as a global config option, was it?  This looks to be documented in the wrong place.  `ts-filter` is just an attribute added on the tables, right?